File: utils/formats.js
const assetsURL = 'https://cdn.zesty.xyz/images/zesty';

const formats = {
    'tall': {
        width: 0.75,
        height: 1,
        style: {
            'standard': `${assetsURL}/zesty-banner-tall.png`,
            'minimal': `${assetsURL}/zesty-banner-tall-minimal.png`,
            'transparent': `${assetsURL}/zesty-banner-tall-transparent.png`
        }
    },
    'wide': {
        width: 4,
        height: 1,
        style: {
            'standard': `${assetsURL}/zesty-banner-wide.png`,
            'minimal': `${assetsURL}/zesty-banner-wide-minimal.png`,
            'transparent': `${assetsURL}/zesty-banner-wide-transparent.png`
        }
    },
    'square': {
        width: 1,
        height: 1,
        style: {
            'standard': `${assetsURL}/zesty-banner-square.png`,
            'minimal': `${assetsURL}/zesty-banner-square-minimal.png`,
            'transparent': `${assetsURL}/zesty-banner-square-transparent.png`
        }
    },
    'mobile-phone-interstitial': {
        width: 0.56,
        height: 1,
        style: {
            'standard': `${assetsURL}/zesty-default-mobile-phone-interstitial.png`,
        }
    },
    'billboard': {
        width: 3.88,
        height: 1,
        style: {
            'standard': `${assetsURL}/zesty-default-billboard.png`,
        }
    },
    'medium-rectangle': {
        width: 1.2,
        height: 1,
        style: {
            'standard': `${assetsURL}/zesty-default-medium-rectangle.png`,
        }
    }
}

const defaultFormat = 'medium-rectangle';
const defaultStyle = 'standard';

export { formats, defaultFormat, defaultStyle };

File: unity/Assets/ZestySDK/Scripts/Internal/Utils/Formats.cs
namespace Zesty
{
    public class Formats
    {
        public enum Types
        {
            MobilePhoneInterstitial,
            Billboard,
            MediumRectangle
        }

        public enum Styles
        {
            Standard,
            Minimal,
            Transparent
        }

        public struct Format
        {
            public Format(double width, double height, string[] images)
            {
                Width = width;
                Height = height;
                Images = images;
            }

            public double Width { get; }
            public double Height { get; }
            public string[] Images { get; }
        }

        // Moible Phone Interstitial
        static string[] mobilePhoneInterstitialImages = {
            $"{Constants.CDN_URL}/images/zesty/zesty-default-mobile-phone-interstitial.png",
        };
        public static Format MobilePhoneInterstitial = new Format(0.56, 1, mobilePhoneInterstitialImages);

        // Billboard
        static string[] billboardImages = {
            $"{Constants.CDN_URL}/images/zesty/zesty-default-billboard.png",
        };
        public static Format Billboard = new Format(3.88, 1, billboardImages);

        // Medium Rectangle
        static string[] mediumRectangleImages = {
            $"{Constants.CDN_URL}/images/zesty/zesty-default-medium-rectangle.png",
        };
        public static Format MediumRectangle = new Format(1.2, 1, mediumRectangleImages);

    }
}
